package.json

```diff
@@ -10,9 +10,9 @@
     "prettier": "^1.19.1"
   },
   "scripts": {
-    "install:eslint-config-base": "cd packages/eslint-config-base && npm install",
-    "install:eslint-config-react": "cd packages/eslint-config-react && npm install",
-    "install:prettier-config": "cd packages/prettier-config && npm install",
+    "install:eslint-config-base": "pushd packages/eslint-config-base; npm install; popd",
+    "install:eslint-config-react": "pushd packages/eslint-config-react; npm install; popd;",
+    "install:prettier-config": "pushd packages/prettier-config; npm install; popd;",
```

Comment:
Interesting, did you have any problems with previous commands?

Reply:
No but directory stack is safer.

Reply:
I found that if I add another package I have to add a command here. I do not really like that so I automated it.

I am afraid my automation would not work on Windows. We have to agree on something.

Reply:
I don't care about windows. It's fine with me.